lte/gateway/c/oai/tasks/mme_app/mme_app_context.c

```diff
@@ -2473,6 +2473,31 @@ static void _mme_app_handle_s1ap_ue_context_release(
   OAILOG_FUNC_OUT(LOG_MME_APP);
 }
 
+bool is_mme_ue_context_network_access_mode_packet_only (
+  mme_ue_s1ap_id_t       mme_ue_s1ap_id)
+{
+  // Function is used to check the UE's Network Access Mode received in ULA from HSS
+
+   struct ue_mm_context_s *ue_context_p = NULL;
+   network_access_mode_t network_access_mode = NAM_MAX;
+
+   OAILOG_FUNC_IN (LOG_MME_APP);
+   ue_context_p = mme_ue_context_exists_mme_ue_s1ap_id(
```

Comment:
Abbas, instead of passing mme_ue_s1ap_id, can we pass the ue_context_p, so that lock and unlock of ue contexts not reqiured

Reply:
sure